addons/project_todo/static/tests/tours/project_todo_main_functions.js

```diff
@@ -123,8 +123,9 @@ registry.category("web_tour.tours").add('project_todo_main_functions', {
 {
     trigger: ".o_field_widget[name='user_ids'] input",
     content: "Assign a responsible to your task",
-    run: "click",
-}, {
+    run: "fill test",
```

Comment:
This breaks the tour as it tries to create a new employee called "test" but because of the hasModal check, the tour can't save since the save button is behind a new form window. Maybe change it to "fill marc"?